wasmbinding/stargate_allowlist.go

```diff
@@ -4,9 +4,11 @@ import (
 	wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/controller/types"
```

Comment:
for uniformity please give the import a custom name e.g. `icacontrollertypes` or something

Reply:
fixed